lib/Serialization/SerializedModuleLoader.cpp

```diff
@@ -200,7 +200,10 @@ FileUnit *SerializedModuleLoader::loadAST(
                        isFramework, loadedModuleFile,
                        &extendedInfo);
   if (loadInfo.status == serialization::Status::Valid) {
-    M.setResilienceStrategy(extendedInfo.getResilienceStrategy());
+    // In LLDB always use the default resilience strategy, so IRGen can query
+    // the size of resilient types.
+    if (!Ctx.LangOpts.DebuggerSupport)
+      M.setResilienceStrategy(extendedInfo.getResilienceStrategy());
```

Comment:
Should we add a test for this? We already have a frontend flag to fake debugger support.

Reply:
The answer is of course, yes we should, but do you have any suggestion how this could be tested inside of swift? It is indirectly tested by the LLDB testsuite.

Reply:
The SIL output should be different when passing around a struct by value. You should be able to define a resilient module with a public struct, and then examine the output of -emit-silgen on a client function that takes an argument of that type.

Reply:
I don’t think it’s worth the effort. The lldb test suite fails without this hack, and it’s only temporary. It will go away once lldb can use remote mirrors to calculate sizes of resilient types.